### PR TITLE
WaveBot Case B

### PR DESCRIPTION
**Pull request by ssolson**
## Description
Adds WaveBot Case B for the new WecOptTool file structure. 

image

## Checklist:

- [x ] Added the results of running the test suite (in pdf form)
- [x ] All new files contain the GPL header
- ~~[ ] If `examples/RM3/optimization.m` has been modified, the content / line 
  numbers in `docs/user/optimization.rst` are still valid or have been fixed~~

[test_results.pdf](https://github.com/SNL-WaterPower/WecOptTool/files/5064316/test_results.pdf)

### CODE REVIEW

**Comment by ryancoe** [comment]
@ssolson - is this ready for review?

**Comment by ssolson** [comment]
In the sense that I need to switch gears for the week and its pretty close yes. As I show the results are different from the paper and I have not investigated why yet. I would love for you to take a look at why the results are different.

File: examples/WaveBot/WaveBot_caseB.m [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+fp = wp/(2*pi);
+Tp = 1/fp; 
+SS = WecOptTool.SeaState.regularWave(w,[A,Tp]);
+
+%% set up device types
+controlType{1} = 'CC';
+controlType{2} = 'P';
+controlType{3} = 'PS';
+controlParams(3).params = {1e10 2e3}; % {zmax, Fmax}
+zmax = 1e10;
```
Comment (ryancoe) [inline examples/WaveBot/WaveBot_caseB.m:54]: should be `zmax = 0.6`

File: examples/WaveBot/WaveBot_caseB.m [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+wp = w(6);
+fp = wp/(2*pi);
+Tp = 1/fp; 
+SS = WecOptTool.SeaState.regularWave(w,[A,Tp]);
+
+%% set up device types
+controlType{1} = 'CC';
+controlType{2} = 'P';
+controlType{3} = 'PS';
+controlParams(3).params = {1e10 2e3}; % {zmax, Fmax}
```
Comment (ryancoe) [inline examples/WaveBot/WaveBot_caseB.m:53]: `controlParams` is no longer used

File: examples/WaveBot/WaveBot_caseB.m [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+zmax = 1e10;
+fmax = 2e3;
```
Comment (ryancoe) [inline examples/WaveBot/WaveBot_caseB.m:55]: set `fmax` to some _large number_, e.g., `fmax=`1e10`

**Comment by ssolson** [comment]
Superceeded by #163

**Pull request closed without merging**